client/my-sites/plans-features-main/index.jsx

```diff
@@ -216,7 +217,7 @@ export class PlansFeaturesMain extends Component {
 			redirectToAddDomainFlow,
 		} = this.props;
 
-		const plans = this.getPlansForPlanFeatures();
+		const plans = this.getPlansForPlanFeatures( experimentVariation );
```

Comment:
I think this change is not needed since `getPlanFeatures()` is used to render the /plans page in Calypso and is not used for the launch flow.

Reply:
Unfortunately, if I remove this change, the experiment will no longer apply on mobile and eCommerce will be visible.

Reply:
Ah yes, you're right. My bad :)